app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @post = @current_user.posts.find(params[:post_id])
```

Comment:
- [ ] Please change the new method logic to be the standard RESTful new method. The post will be identified in the create method
`@comment = Comment.new`